src/ChinookSnippets/ChinookSnippets.csproj

```diff
@@ -152,10 +152,10 @@
     </None>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.VisualStudio.SDK" Version="17.0.31902.203" ExcludeAssets="runtime">
```

Comment:
is updating the packages version necessary?

Reply:
I would do it, if it still works.